x-pack/plugins/security_solution/public/flyout/left/header.tsx

```diff
@@ -44,14 +46,15 @@ export const PanelHeader: VFC<PanelHeaderProps> = memo(({ selectedTabId, setSele
     ));
 
   return (
-    <EuiFlyoutHeader hasBorder>
-      <EuiTabs
-        size="l"
-        expand
-        css={css`
-          margin-bottom: -25px;
-        `}
-      >
+    <EuiFlyoutHeader
+      hasBorder
+      css={css`
+        background-color: ${euiTheme.colors.lightestShade};
+        padding-bottom: 0 !important;
```

Comment:
this removes the padding at the bottom to follow the mocks (and keep the UI we have currently), or it would look like this
![Screenshot 2023-09-22 at 5 59 48 PM](https://github.com/elastic/kibana/assets/17276605/c2becd8d-ff3b-4475-8692-f192bd77bc67)